src/main/kotlin/io/github/zap/build/gradle/convention/ProjectExtension.kt

```diff
@@ -169,4 +171,99 @@ fun Project.configureShadow(destination: File,
     tasks.named("build", DefaultTask::class.java) {
         it.dependsOn(sj)
     }
+}
+
+fun Project.qs (name: String = "", options: ExternalModuleDependency.() -> Unit = {}) : ExternalModuleDependency.() -> Unit {
```

Comment:
space between method name and paranthesis? literally unacceptable
also you should use kotlin-style line-wrapping for long method signatures if we are working in Kotlin

Reply:
why not 😶‍🌫️